### DISCUSSION TITLE
Social media preview cards for members

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#18373] [Feature Requests]
I like promoting community members across the socials, in order to do this more effectively it would be cool to able to share a member profile with a cool social media card. I've attached a picture of a cute rabbit to make this even more appealing and to bribe someone when it comes to discussing pipeline priorities.

![rabbit](https://user-images.githubusercontent.com/12694270/186464642-14c97212-f10e-40cb-bcb5-9761fdb7dfc2.gif)

**Comment by amywtlin** [comment]
Can't guarantee anything yet, but +1 point for the cute pic!

@leewynne seriously, though. Can you share a few examples of these social previews you think would be cool, if you happen to have some handy?

**Reply by leewynne** [reply]
Hey Amy, thanks for the point ! 🎉

I think this is probably a simple example. 
<img width="526" alt="Screenshot 2022-09-05 at 15 59 38" src="https://user-images.githubusercontent.com/12694270/188477617-8094093b-651d-4826-9d9b-38539c4fc87c.png">
